File: week-01/임영빈/01/solution.cpp
#include <vector>
#include <string>
#include <unordered_map>

using namespace std;
 
string solution(vector<string> participant, vector<string> completion)
{
    string answer = "";
    unordered_map<string, int> hash_map;
    for (string key : participant)
    {
        hash_map[key]++;
    }
    for (string key : completion)
    {
        hash_map[key]--;
    }
    for (auto pair : hash_map)
    {
        if (pair.second > 0)
        {
            answer = pair.first;
            break;
        }
    }
    return answer;
}

File: week-01/임영빈/01/solution.py
def solution(participant, completion):
	hash_map = dict()
	for i in participant:
		hash_map[i] = hash_map.get(i, 0) + 1
	for j in completion:
		hash_map[j] -= 1
	for answer in hash_map:
		if hash_map[answer]:
			return answer